ennead/views/admin.py

```diff
@@ -0,0 +1,64 @@
+"""Views, used for task creation and editing"""
+
+from flask import abort, redirect, render_template, request, url_for
+from werkzeug.wrappers import Response
+
+from ennead.utils import require_teacher
+from ennead.models.task import Task, TaskSet
+
+
+@require_teacher
+def adm_task_list_page() -> Response:
+    """GET /adm/tasks: list of tasks & creation of new"""
+
+    return render_template('adm_task_list.html', task_set=TaskSet.get(active=True))
+
+
+@require_teacher
+def task_edit_page(task_id: int) -> Response:
+    """GET /adm/tasks/<task_id>: task edit page"""
+
+    try:
+        task = Task.get(Task.id == task_id)
+    except Task.DoesNotExist:
+        abort(404)
+
+    return render_template('task_edit.html', task=task)
+
+
+@require_teacher
+def task_edit() -> Response:
+    """POST /adm/tasks: create or update task"""
+
+    if request.form['id']:
+        try:
+            task = Task.get(Task.id == int(request.form['id']))
```

Comment:
`task = Task.get_by_id(request.form['id'])`

Reply:
```suggestion
            task = Task.get_by_id(int(request.form['id']))
```
Можешь сразу отправлять suggestion в следующий раз. :)